engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
 :::
 
 (meetings:backlog-refinement)=
+
 ### 1. Pre-planning and refinement
 
-The Refinement meetings prepares the Iteration board for the Iteration planning
-meetings, which relies on having sequenced and prioritized tasks to be available in the board's Refined column.
+Tasks in the backlog are continually added and refined asynchronously by the whole team, ready for Iteration planning meetings, which rely on having sequenced and prioritized tasks to be available in the board's Refined column.
+The backlog is regularly culled of tasks that are stale, in a monthly Backlog refinement meeting 
 
-Backlog refinement meetings should drive:
-- creation of tasks into the sprint board's backlog column
-- promotion of tasks into the sprint board's refined column
-- prioritization of task in the sprint board's refined column
+Backlog refinement meetings should aim to
+- Identify and remove tasks that are stale or obsolete
+- Identify and action tasks that are in the wrong status or priority on the board
+- Identify tasks that need further asynchronous refinement
 
-Tasks should tie back to initiatives, and the refined column should also function as a buffer of uncommitted work to pull from if needed.
+Tasks should tie back to initiatives or contract deliverables, and the refined column should also function as a buffer of uncommitted work to pull from if needed.
 
 Attendees of the Refinement meeting should include representatives from the following teams:
 - Engineering: for technical decision-making
 - Product: for product decision-making
-- Partnerships and Community Success: for community/grant decision-making.
+
+Product Managers at the meeting should ensure to come prepared to represent Business Development interests.
 
 (meetings:sprint-planning)=
-### 2. Iteration Planning
 
-The team conducts a synchronous iteration planning meeting for 60 minutes at the beginning of each iteration.
+### 2. Iteration Planning Guide
+
+#### 🗓️ Between Refinement and Planning
+
+A Product Manager will:
+
+- **Add deadlines to issues that need them**  
+  A Product Manager will ensure that any issue with a contractual obligation or expected delivery has a clear due date.
+
+- **Mark contract deliverables with dates**  
+  Use labels to easily identify Contract Deliverables.
+
+- **Assign preferred sprint-end dates**  
+  If a card is intended for the upcoming sprint, let a Product Manager know and they will mark it with this sprint’s end date.
+
+- **Organize in 'Up Next'**  
+  A Product Manager manages and sorts cards in the ‘Up Next’ column for visibility.
+
+The Team will:
+
+- **Review open-source contributions**  
+  Include relevant contributions and initiatives in planning discussions.
```

Comment:
Can we find a specific role to lead this? Our upstream fixes have vanished for a few months now.